cgmanifest.json

```diff
@@ -7915,8 +7915,8 @@
         "type": "other",
         "other": {
           "name": "selinux-policy",
-          "version": "2.20210203",
-          "downloadUrl": "https://github.com/SELinuxProject/refpolicy/releases/download/RELEASE_2_20210203/refpolicy-2.20210203.tar.bz2"
+          "version": "2.20200818",
```

Comment:
This looks suspicious. Should this change be here?

Reply:
Nope! Was a relic of me testing locally. Fixed.